src/main/java/edu/iu/uits/lms/canvasnotifier/config/LmsAuthorizationEventPublisher.java

```diff
@@ -0,0 +1,27 @@
+package edu.iu.uits.lms.canvasnotifier.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.security.authorization.AuthorizationDecision;
+import org.springframework.security.authorization.AuthorizationEventPublisher;
+import org.springframework.security.authorization.event.AuthorizationEvent;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+import java.util.function.Supplier;
+
+@Component
+@Slf4j
+public class LmsAuthorizationEventPublisher implements AuthorizationEventPublisher {
+    private ApplicationEventPublisher applicationEventPublisher;
```

Comment:
Maybe we put this publisher in a common place (vs in the tool here) like the old stuff was? The issue doing that is that anything using that common place will be publishing authorization events. And, perhaps, that's okay?